client/gameObjects/Skeleton.js

```diff
@@ -189,7 +190,9 @@ export default class Skeleton extends Phaser.GameObjects.Image {
     this.setAlphaTiles();
 
     // @TODO: Implement fist fighting if there is no weapon worn #171
-    const range = this.scene.equipment.weapon === "sword" ? 1 : 5;
+    const currentWeapon =
+      GAME_ITEMS[this.scene.equipment && this.scene.equipment.weapon];
```

Comment:
What 😄 sometimes `this.scene.equipment` is undefined so there was an error while trying to read property from undefined.
It will be refactored and fixed with #171